ansible_files/roles/oci/setup_oci_for_test_infra/templates/assisted-additional-config.j2

```diff
@@ -1,3 +1,5 @@
+export CLUSTER_NAME=test-infra-oci-{{ lookup('community.general.random_string', special=false, upper=false) }}
+
 export PLATFORM=external
 export TF_PLATFORM=oci
```

Comment:
remove

Reply:
approved the PR, do you want to remove it in this PR or separate one ?

Reply:
yes